src/services/tfjs-service.ts

```diff
@@ -79,8 +67,20 @@ const linearRegression = async ({model: modelRef, training, query, fits}) => {
   return buffer.values;
 };
 
+
+export const loadImage = async (url: string): Promise<HTMLImageElement> => {
```

Comment:
I mentioned this above (and this whole thing is minor) but this isn't a good place for this utility, if for no other reason then importing causes Service to be registered (which happens anyway in your use-case, but in the fullness of time...).

Reply:
e4a9294